pkg/controller/backup_controller.go

```diff
@@ -148,12 +148,36 @@ func NewBackupController(
 }
 
 func (c *backupController) resync() {
+	// recompute backup_total metric
 	backups, err := c.lister.List(labels.Everything())
 	if err != nil {
 		c.logger.Error(err, "Error computing backup_total metric")
 	} else {
 		c.metrics.SetBackupTotal(int64(len(backups)))
 	}
+
```

Comment:
Error message on line#154 seems out of place, no?

Reply:
how so? In any case, unrelated to my PR :)

Reply:
Yes, totally unrelated to this PR.